### PR TITLE
doc: update the readme.md file with latest UI

### LINKED ISSUE #4: update: update README.md file with latest UI
currently README.md file contains old UI screenshots. update the [README.md ](https://github.com/malivinayak/Tic-Tac-Toe#readme)File with appropriate details.

### PR DESCRIPTION
**Pull request by SumitMangrati**
the readme.md file has been updated with the latest ui image 

Resolves #4

### CODE REVIEW

File: README.md
Comment (malivinayak) [inline README.md:8]: can you delete the extra new line

**Review by malivinayak — CHANGES_REQUESTED** [review]
Nicely Done, just a small change

File: README.md
Comment (malivinayak) [reply README.md:8]: If not then its ok. As you are new to open source I will directly approve the changes and merge the PR

**Review by malivinayak — APPROVED** [review]
Perfect

File: README.md
Comment (SumitMangrati) [reply README.md]: okay ill work to remove the extra lines

**Comment by malivinayak** [comment]
@SumitMangrati you are working on issue #4 

So in the PR description simple mention `Resolves #4` at the end
This will join this PR to that issue.

and after merging this PR, respective issue will automatically close

Try editing the description and I will merge the PR

**Comment by malivinayak** [comment]
> @SumitMangrati you are working on issue #4
> 
> So in the PR description simple mention `Resolves #4` at the end This will join this PR to that issue.
> 
> and after merging this PR, respective issue will automatically close
> 
> Try editing the description and I will merge the PR

I am merging the PR
Just remember that you can link issue to PR y mentioning `Resolves #[issue No.]`

**Comment by malivinayak** [comment]
@SumitMangrati You are trying to change title
image


and I am talking about description
image


I have updated it just now

**Comment by SumitMangrati** [comment]
could you check it once i have worked on removing the extra lines

**Review by malivinayak — APPROVED** [review]

**Comment by malivinayak** [comment]
image

Here you can see that after adding `Resolves #4` it automatically linked the issue

**Comment by malivinayak** [comment]
> could you check it once i have worked on removing the extra lines

Yes, you have done it perfectly

**Comment by SumitMangrati** [comment]
thank you now i understood by mentioning the number my pr gets directly linked to that particular issue

**Comment by malivinayak** [comment]
> thank you now i understood by mentioning the number my pr gets directly linked to that particular issue

Yes.
I will merge the PR
Hope you learn something new😊

**Comment by SumitMangrati** [comment]
Thank you so much I learned a lot from you. Started my journey of open source with you. In the Hacktober tab it says my PR needs to under 7 days reviews. Thank you

**Comment by malivinayak** [comment]
> Thank you so much I learned a lot from you. Started my journey of open source with you. In the Hacktober tab it says my PR needs to under 7 days reviews. Thank you

Yes after 7 days it will automatically goes to accepted. It is one of the rule of hacktoberfest
They wait for 7 days to check if repo maintainers keep the PR or remove.

Don't worry about this PR. I have accepted it

**Comment by malivinayak** [comment]
You can connect me at [LinkedIn](https://www.linkedin.com/in/malivinayak/)

**Pull request merged**
